deployment/docker/sample.env

```diff
@@ -149,6 +149,9 @@ REDIS_ENABLED=false
 CUSTOM_SUBMISSION_ENABLED=false
 CUSTOM_SUBMISSION_URL=http://{your-ip-address}:{port}
 
+#Camunda - History Level Configuration
```

Comment:
Can you provide other values as example; where user can disable this to enhance performance ?